### PR TITLE
[REF] point_of_sale: refactor unit tests to hoot

### PR DESCRIPTION
**Pull request by vlst-odoo**
In this commit we refactor the pos unit tests from qunit to hoot.

https://github.com/odoo/enterprise/pull/69008
---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/core/browser/feature_detection.js [hunk 1]
```diff
@@ -29,10 +29,11 @@ export function isAndroid() {
...
 
 export function isIOS() {
-    return (
-        /(iPad|iPhone|iPod)/i.test(browser.navigator.userAgent) ||
-        (browser.navigator.platform === "MacIntel" && maxTouchPoints() > 1)
-    );
+    // return (
+    //     /(iPad|iPhone|iPod)/i.test(browser.navigator.userAgent) ||
+    //     (browser.navigator.platform === "MacIntel" && maxTouchPoints() > 1)
+    // );
```
Comment (caburj) [inline addons/web/static/src/core/browser/feature_detection.js:35]: ?

Comment (vlst-odoo) [reply addons/web/static/src/core/browser/feature_detection.js]: my bad. It's reverted now

**Comment by caburj** [comment]
robodoo r+

**Comment by caburj** [comment]
robodoo rebase-ff

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**